### ISSUE TITLE
How much GPU memory is used?

### ISSUE THREAD

**Issue by kxhit** [JenningsL/PointRCNN#4]
Hi! I set the sa_module as [4096, 1024, 256, 64] and get GPU memory exhausted error. I'm using TITAN XP 12G GPU. I want to know how much GPU memory is used? 

I have not tried the official version PointRCNN yet (implemented by pytorch). [However the author told me 12GB memory GPU is enough](https://github.com/sshaoshuai/PointRCNN/issues/1). I don't know if I made some mistakes or it is due to pytroch/tensorflow version. 

Thanks! Waiting for your reply!

**Comment by JenningsL** [comment]
Yes, I use TITAN V with 12G memory. It's enough for batch size=4.

**Comment by kxhit** [comment]
@JenningsL I could run the code using batch_size=4, though batch_size is 16 in official version. Thanks for your prompt reply!
